### PR TITLE
Bugfix/#18 scroll doesn't work in search tab ios

### LINKED ISSUE #18: Iphone - scroll doesn't work in search tab
## Current behavior

Image from iOS
User can't scroll in search tab on iphone

## Can you handle fixing this bug by yourself?

- [ ] YES
- [ ] NO

## Environment details

- Browser: 
- OS: 
- Code Version:

### PR DESCRIPTION
**Pull request by lukaszjedrasik**
### Related Issues

Closes #18 

### Short Description and Why It's Useful
I have moved `disableBodyScroll(this.$el)` and `clearAllBodyScrollLocks()` to `watch` and I have used `windowHelper` to fix the scrolling issue in Search Panel on iOS.

#### Before
https://user-images.githubusercontent.com/42383376/83997922-cd535800-a95f-11ea-8830-2a6009eb1deb.png

#### After

4ly4q3

### Contribution and Currently Important Rules Acceptance
- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-default/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: components/theme/blocks/AsyncSidebar/AsyncSidebar.vue [hunk 1]
```diff
@@ -94,7 +94,7 @@ export default {
 .right-sidebar {
   top: 0;
   right: 0;
-  z-index: 4;
+  z-index: 1001;
```
Comment (Fifciu) [inline components/theme/blocks/AsyncSidebar/AsyncSidebar.vue:97]: Couldn't we have a lower value?

**Review by Fifciu — CHANGES_REQUESTED** [review]

File: components/theme/blocks/AsyncSidebar/AsyncSidebar.vue (hunk 1 above)
Comment (lukaszjedrasik) [reply components/theme/blocks/AsyncSidebar/AsyncSidebar.vue:97]: Back to top have `z-index: 1000`

Zrzut ekranu 2020-11-12 o 11 40 40

**Review by gibkigonzo — APPROVED** [review]

**Review by Fifciu — APPROVED** [review]

**Comment by Fifciu** [comment]
Please resolve conflict in changelog @lukaszjedrasik

**Comment by lukaszjedrasik** [comment]
@Fifciu Done

**Pull request merged**